src/deezer/client.py

```diff
@@ -406,7 +412,7 @@ def get_user_flow(self, **kwargs) -> PaginatedList[Track]:
         """
         return PaginatedList(client=self, base_path="user/me/flow", **kwargs)
```

Comment:
In this case we don't fetch the PaginatedList. Is that intended? Feels wrong to me to be honest, especially because the function is async without making use of that

Reply:
```suggestion
        return await self._get_paginated_list("user/me/flow", **kwargs)
```
I think this should be preferred

Reply:
Yeah correct